schemas/yaml/Language.yml

```diff
@@ -375,6 +486,26 @@ examples:
     type:                Language
     url:                 https://data.digitallinguistics.io/languages/Chitimacha
 
+  - name: Shangaji
```

Comment:
I really like the two sets of example data you chose for this! In general, it's always a great idea to test code using realistic data, so this is really nice. Thanks for doing the work of researching these languages for this.